common/static/common/templates/discussion/response-comment-show.underscore

```diff
@@ -1,4 +1,7 @@
-<div id="comment_<%- id %>">
+<div class="discussion-comment" id="comment_<%- id %>">
+  <div class="author-image level-comment">
+    <img class="image-frame" src="https://d1kekzok76m982.cloudfront.net/prod/5d0da5229e4c83894a51e04a41e06cd0_120.jpg?v=1441780917" alt="Profile image for post author">
```

Comment:
Note: Prior to merge, I will remove this img element (if empty, this div is collapsed currently.) I've noted this in the PR checklist as well.